examples/dwarfdump.rs

```diff
@@ -386,7 +387,12 @@ fn main() {
         "print .eh-frame exception handling frame information",
     );
     opts.optflag("G", "", "show global die offsets");
-    opts.optflag("i", "", "print .debug_info and .debug_types sections");
+    opts.optflagopt(
```

Comment:
So what should we do instead?

Reply:
`dwarfdump -i -- filename` works but yeah that's not ideal.

Reply:
How about `-o`/`--offset`?

Reply:
I guess we could do that, though the reason I chose this was to match `llvm-dwarfdump`.

Maybe we should switch to `structopt` instead. I think it can be made to do the right thing here.

Reply:
Ah okay, yeah matching `llvm-dwarfdump` would be good (`llvm-dwarfdump --help` didn't show that option). If you want, can leave this how it is and switch later.